### DISCUSSION TITLE
Provide ctest environment to launch configuration

### DISCUSSION THREAD

**Discussion by MarcelKoller** [microsoft/vscode-cmake-tools#3966] [Ideas]
Would it be possible to provide other test properties than `WORKING_DIRECTORY` (especially `ENVIRONMENT`) to be used in configurations in `launch.json`? Or is this already possible in some way?

**Comment by gcampbell-msft** [comment]
@MarcelKoller Currently I don't believe this is possible, but if you'd like it as a feature request, that'd be great to create an issue on this repo so that we can more closely track it. Thanks!
